### ISSUE TITLE
App needs debugging

### ISSUE THREAD

**Issue by bigsee** [JennHopkins/dressagedancers#1]
@JennHopkins just creating an issue here so we have somewhere to chat. Do you have Discord maybe?

**Comment by JennHopkins** [comment]
I do not have Discord.

**Comment by bigsee** [comment]
No problem. I highly recommend it anyway as there are very good Nuxt and Vue servers where users of all levels help each other with issues.

I think I've found some issues that might be stopping it from compiling anyway. e.g.:

<img width="1288" alt="Screen Shot 2020-08-09 at 20 58 28" src="https://user-images.githubusercontent.com/11768967/89739694-2b490c80-da83-11ea-9b88-c340ea674b36.png">


Looks like you have more than one root element inside one of your templates atm. the one called `horsetwo.vue`.

**Comment by JennHopkins** [comment]
Oh, crikey! Look at that! I'll fix that.

(I didn't want to send you on a wild goose chase - there's probably LOTS of things wrong. Sigh! I hate being new.)

Thanks for your help. :)

**Comment by bigsee** [comment]
No problem at all! Just glad to be able to help...

The first thing I would say is that the nuxt.config.js file looks strange. Did you use the nuxt CLI `create-nuxt-app` to start the project?

Either way, feel free to leave this open and just ping me on here if you get stuck at any point. I'll be happy to help if I can, even if I'm not around immediately. 

Good luck! :)

**Comment by bigsee** [comment]
P.S. We were all new once! Me not so long ago. Keep at it! 

You'll be amazed how soon you'll be looking back at stuff like this and it will feel like the distant past! ^.^

**Comment by JennHopkins** [comment]
No, I didn't use "create-nuxt-app" in order to start the project. I used "vue init nuxt-community/starter-template [project name here]" instead. This was on the advice of the instructor teaching the Front End Dev course I took. He assured us that it would work better than...I forgot what the alternative was, but "create-nuxt-app" must have been it.

The thing about that cat is that he wasn't good about telling us the whys or the wherefores of Front End, just told us, "do this, do that." And if we had questions, he'd answer obtusely. So, I have no idea why he wanted us to use that other command. So, "create-nuxt-app", I'll give it try. What the heck. Thanks! :)

**Comment by bigsee** [comment]
I hear ya! It's tough sometimes when you don't get quite as much detail as you might like from an instructor. 

I always go use Nuxt's create package. You can find it [here](https://github.com/nuxt/create-nuxt-app).

The command is `npx create-nuxt-app <your-project>`.
